phoenicis-repository/src/test/java/org/phoenicis/repository/repositoryTypes/CachedRepositoryTest.java

```diff
@@ -38,8 +38,8 @@ public void testFetchInstallableApplications() throws Exception {
                         .thenReturn(
                                 new RepositoryDTO.Builder()
                                         .withCategories(
-                                                Arrays.asList(new CategoryDTO.Builder().withName("Category 1").build(),
-                                                        new CategoryDTO.Builder().withName("Category 2").build()))
```

Comment:
In fact we can still use a name and it is easier. It is just that it needs to be translated when it is used in a label